packages/lint-repository-sui/src/rules/sui-tools-version.js

```diff
@@ -0,0 +1,88 @@
+const dedent = require('string-dedent')
+
+const SUI_TOOLS_VERSIONS = {
```

Comment:
Maybe we could think of checking the versions defined are the `latest`... but we can move on like this for now 👌